app/src/main/java/com/example/datarsd1/icebreaker/QuizActivity.java

```diff
@@ -0,0 +1,207 @@
+ver 1.0-
+
+QuizActivity.java
+
+Java activity file for quiz activity of IceBreaker -
+IceBreaker is a MDB member trivia quiz designed to allow new members to quickly
+learn more about MDB members from previous semesters.
+
+The quiz activity (homescreen) of icebreaker contains the game (questions+responses)
+and outlines most of the application functionality
+
+possible changes: use a scrollview to make quiz playable on horizontal mode. (OR lock app
+in vertical mode for play)
+Remove quiz activity and replace with quiz fragment (combine 2 current activities)
+implement customview to display all quiz info instead of using separate views for options,
+images, textviews, etc.
+
+ */
+
+package com.example.datarsd1.icebreaker;
+
+import android.content.Intent;
+import android.os.CountDownTimer;
+import android.provider.ContactsContract;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+import java.util.Arrays;
+
+public class QuizActivity extends AppCompatActivity implements View.OnClickListener{
+
+    private Button b1,b2,b3,b4;
+    private TextView t1;
+    private String answer;
+    private int score;
+    private CountDownTimer timer;
+    private boolean cheat;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_quiz);
+
+        answer = "";
+        score = 0;
+        cheat = getIntent().getExtras().getBoolean("CHEAT");
+
+        String[] members = {"Jessica Cherny", "Kevin Jiang", "Jared Gutierrez", "Kristin Ho", "Christine Munar", "Mudit Mittal", "Richard Hu", "Shaan Appel", "Edward Liu", "Wilbur Shi", "Young Lin", "Abhinav Koppu", "Abhishek Mangla",  "Akkshay Khoslaa", "Ally Koo", "Andy Wang", "Aneesh Jindal", "Anisha Salunkhe", "Aparna Krishnan", "Ashwin Vaidyanathan", "Cody Hsieh", "Jeffrey Zhang", "Justin Kim", "Krishnan Rajiyah", "Lisa Lee", "Peter Schafhalter", "Sahil Lamba", "Sameer Suresh", "Sirjan Kafle", "Tarun Khasnavis", "Billy Lu", "Aayush Tyagi", "Ben Goldberg", "Candice Ye", "Eliot Han", "Emaan Hariri", "Jessica Chen", "Katharine Jiang", "Kedar Thakkar", "Leon Kwak", "Mohit Katyal", "Rochelle Shen", "Sayan Paul", "Sharie Wang", "Shreya Reddy", "Shubham Goenka", "Victor Sun", "Vidya Ravikumar"};
+        Arrays.sort(members, String.CASE_INSENSITIVE_ORDER);
+        QuestionGenerator.setNames(members);
+
+        b1 = (Button) findViewById(R.id.option1);
+        b1.setOnClickListener(this);
+        b2 = (Button) findViewById(R.id.option2);
+        b2.setOnClickListener(this);
+        b3 = (Button) findViewById(R.id.option3);
+        b3.setOnClickListener(this);
+        b4 = (Button) findViewById(R.id.option4);
+        b4.setOnClickListener(this);
```

Comment:
No need to separate these two lines, can say findViewById(R.id.option1).setOnClickListener(this), also casts are irrelevant for onclicklisteners